documentation/library-reference/source/c-ffi/index.rst

```diff
@@ -75,6 +98,11 @@ by the class ``<C-double>``, and ``unsigned long`` by the class
    Dylan names appearing in this document, is not binding and can safely be
    ignored.
 
+.. index::
+   single: <C-double*> class
```

Comment:
Does the "*" here need to be escaped or does it show up okay in the index?  I only noticed it because GitHub's diff display gets confused by it.

Reply:
It shows OK in the index.